packages/sdk-core/src/ConstantFlowAgreementV1.ts

```diff
@@ -237,6 +309,233 @@ export default class ConstantFlowAgreementV1 {
+
+        const callData = cfaInterface.encodeFunctionData(
+            "authorizeFlowOperatorWithFullControl",
+            [normalizedToken, normalizedSender, normalizedFlowOperator, "0x"]
+        );
+
+        return this.host.populateCallAgreementTxnAndReturnOperation(
+            this.options.config.cfaV1Address,
+            callData,
+            userData,
+            overrides
+        );
+    }
+
+    /**
+     * @dev Revoke flow operator control - set flow rate to 0 with no permissions.
+     * @param superToken The token to be flowed.
+     * @param sender The sender of the flow.
+     * @param flowOperator The permission grantee address
+     * @param userData Extra user data provided.
+     * @param overrides ethers overrides object for more control over the transaction sent.
+     */
+    revokeFlowOperatorWithFullControl({
+        superToken,
+        sender,
+        flowOperator,
+        userData,
+        overrides,
+    }: IFullControlParams): Operation {
+        const normalizedToken = normalizeAddress(superToken);
+        const normalizedSender = normalizeAddress(sender);
+        const normalizedFlowOperator = normalizeAddress(flowOperator);
+
+        const callData = cfaInterface.encodeFunctionData(
+            "revokeFlowOperatorWithFullControl",
+            [normalizedToken, normalizedSender, normalizedFlowOperator, "0x"]
+        );
+
+        return this.host.populateCallAgreementTxnAndReturnOperation(
+            this.options.config.cfaV1Address,
+            callData,
+            userData,
+            overrides
+        );
+    }
+
+    /**
+     * @dev Create a flow as an operator
+     * @param flowRate The specified flow rate.
+     * @param sender The sender of the flow.
+     * @param receiver The receiver of the flow.
+     * @param superToken The token to be flowed.
+     * @param userData Extra user data provided.
+     * @param overrides ethers overrides object for more control over the transaction sent.
+     * @returns {Operation} An instance of Operation which can be executed or batched.
+     */
+    createFlowByOperator = ({
+        flowRate,
+        sender,
+        receiver,
+        superToken,
+        userData,
+        overrides,
+    }: ICreateFlowParams): Operation => {
+        const normalizedToken = normalizeAddress(superToken);
+        const normalizedReceiver = normalizeAddress(receiver);
+        const normalizedSender = normalizeAddress(sender);
+
+        const callData = cfaInterface.encodeFunctionData(
+            "createFlowByOperator",
+            [
+                normalizedToken,
+                normalizedSender,
+                normalizedReceiver,
+                flowRate,
+                "0x",
+            ]
+        );
+
+        return this.host.populateCallAgreementTxnAndReturnOperation(
+            this.options.config.cfaV1Address,
+            callData,
+            userData,
+            overrides
+        );
+    };
```

Comment:
A lot of these function could _technically_ be turned into a shorter syntax (not yet saying we should do it):
```ts
createFlowByOperator = ({
    flowRate,
    sender,
    receiver,
    superToken,
    userData,
    overrides,
}: ICreateFlowParams): Operation =>
    this.host.populateCallAgreementTxnAndReturnOperation(
        this.options.config.cfaV1Address,
        cfaInterface.encodeFunctionData("createFlowByOperator", [
            normalizeAddress(superToken),
            normalizeAddress(sender),
            normalizeAddress(receiver),
            flowRate,
            "0x",
        ]),
        userData,
        overrides
    );
```